UPGRADE.md

```diff
@@ -0,0 +1,7 @@
+This document describes future changes that will infect your current resources used inside of OpenShift.
+Each change contains description of the change and a date when that change will happen.
+
+* Build's pod label will be renamed from `build` to `io.openshift.build.name`.
+  All the newly created Builds will already have the new label, but old ones will
+  be recognized (inside the build controller) until post 3.1/1.2 version.
```

Comment:
Add an H2/H3 for the version this applies to (1.0.0) and then each item should be part of a list.  We should list as "OSE 3.1" and "Origin 1.x" when we know the Xs